app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -235,6 +250,7 @@ class TabsTrayFragment : AppCompatDialogFragment() {
             trayInteractor = tabsTrayInteractor,
             browserInteractor = browserTrayInteractor,
             navigationInteractor = navigationInteractor,
+            inactiveTabsInteractor = inactiveTabsInteractor,
```

Comment:
The amount of interactors that we are passing around is kinda exhausting when I look at it. It would be nice if we should just do something like https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlInteractor.kt#251. It still nice that we have interactors for individual feature/components, but ideally we should just pass one down. 

This could be a followup if we feel strongly about it.

Reply:
Are you basically suggesting to consolidate all of them into one top-level TabsTrayInteractor, so only the one is passed around?

Reply:
Yes in effect